cfgov/jinja2/v1/_includes/molecules/hero.html

```diff
@@ -6,13 +6,59 @@
 
    Description:
 
-   Create a hero molecule. See [GHE]/flapjack/Modules-V1/wiki/Heros
+   Create a hero molecule.
+
+   settings:             An object with the following options for settings.
+
+   REQUIRED:
+
+   settings.heading:     A string containing title of module.
+
+   settings.body:        A string containing the content to display below the
+                         heading (typically a paragraph).
+
+   settings.image_url:  A string containing the url of the image to be placed
+                         adjacent to the text
+
+   OPTIONAL:
+
+   settings.background: A string containing the color code (in any format) to
+                         set as the hero background color
+
+   settings.links:       A tuple of items to create a list of links.
```

Comment:
Shouldn't this be `settings.link_url` and `settings.link_text`? It doesn't seem like `settings.links` is used or exists?